src/zmq/CMakeLists.txt

```diff
@@ -12,13 +12,10 @@ add_library(bitcoin_zmq STATIC EXCLUDE_FROM_ALL
 target_compile_definitions(bitcoin_zmq
   INTERFACE
     ENABLE_ZMQ=1
-  PRIVATE
-    $<$<AND:$<PLATFORM_ID:Windows>,$<CXX_COMPILER_ID:GNU>>:ZMQ_STATIC>
 )
 target_link_libraries(bitcoin_zmq
   PRIVATE
     core_interface
+    zeromq
```

Comment:
Doubtful it matters here, but external libs after internal libs please.

Reply:
Sure. Reordered.